### ISSUE TITLE
Allow bulk writes

### ISSUE THREAD

**Issue by Al2Klimov** [nrdcg/desec#5]
... to combine several modifications in one request (which seem to be rate-limited btw., at least this lib rate-limits them).

I.e.: https://desec.readthedocs.io/en/latest/dns/rrsets.html#modifying-an-rrset

**Comment by adamhassel** [comment]
Would that not simply be changing a function signature or two?

Like

```golang
func (s *RecordsService) Update(ctx context.Context, domainName, subName, recordType string, rrSet RRSet) (*RRSet, error)
```
becomes 
```golang
func (s *RecordsService) Update(ctx context.Context, domainName, subName, recordType string, rrSet ...RRSet) ([]RRSet, error)
```
and then you update the test and the docs, and Bob's your uncle?

**Comment by ldez** [comment]
Fixed by #9
